cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -2177,7 +2177,7 @@ Slice::Python::CodeVisitor::writeDocstring(const OperationPtr& op, DocstringMode
     //
     // Emit the general description.
     //
-    _out << nl << "\"\"\"";
+    _out << nl << R"(""")";
```

Comment:
why not use `tripleQuotes` here and the spot below

Reply:
Fixes (2 spots)